tests/client.py

```diff
@@ -15,6 +15,7 @@ class TestClient():
     START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z"
     V3_DEALS_PROPERTY_PREFIXES = {'hs_date_entered', 'hs_date_exited', 'hs_time_in'}
     BOOKMARK_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+    record_create_times = {}
```

Comment:
I don't see where we use this.  Can you point it out to me?

I see it now in the test. I was expecting it to be used in the client. See comment below.